UI/MapUI.gd

```diff
@@ -2,17 +2,18 @@ extends Control
 
 var map_scene: PackedScene = preload("res://#Scenes/CardScrollUI.tscn")
 var room_ui: PackedScene = load("res://Map/RoomUI.tscn")
-var _padding_offset = 20
-var _MINIMUM_ROOM_WIDTH = 510
-var _MINIMUM_ROOM_HEIGHT = 490
-var _LIGHT_FLOOR_RANGE = 3
+var _padding_offset: float = 25
+var _MINIMUM_ROOM_WIDTH: float = 510
+var _MINIMUM_ROOM_HEIGHT: float = 490
+var _LIGHT_FLOOR_RANGE: int = 3
 
 @export var color_rect: ColorRect
 @export var scroll_container: SmoothScrollContainer
 @export var room_container: ColorRect
 @export var room_addition_node: Control
 @export var torch_confirmation_dialog: ConfirmationDialog
-@export var test_torch_indices: Dictionary
+@export var test_torch_indices: Array[Vector2]
+@export var test_player_position: Vector2
```

Comment:
this can be a `Vector2i` since positions will be on the map and that's just integers